news/2024/2024-03-19-bespoke-music-in-osu.md

```diff
@@ -0,0 +1,259 @@
+> 
+> After wrapping up the [first edition of the osu!taiko Malaysian Tournament](https://osu.ppy.sh/community/forums/topics/1389964?n=1) (OTMT) series back in 2021, the team didn't want to just do the same thing again for the next year and beyond, so we felt that the most natural progression for the series' growth was to start introducing original songs into the mix.
+> 
+> I truly believe that this, along with the introduction of our mascot character, **IRIS**, helped a lot in weaving a recognizable visual identity for the whole tournament, which significantly enhanced the tournament experience.
+> 
+> ![osu!taiko Malaysia mascot "IRIS"](/wiki/shared/news/2024-03-19-bespoke-music-in-osu/iris.jpg)
+
+Visual arts go hand in hand with music commissions and tournament production, which is itself an interesting topic (that we may cover in the near future!).
+
+> The music production process for our team is rather straightforward. We'll usually only specify the particular genre and difficulty that we're looking for to an artist based on the style that they're comfortable with, along with some example tracks provided. Besides that, the artists are given creative liberties to elevate it however they wish.
+> 
+> I think this process works very well as we also have a general channel for all our composers to mingle and share ideas for each other's works, allowing for more constructive opinions to be provided throughout the process by equally skilled individuals.
+
+---
+
+The Malaysian tournaments collectively have 19 original tracks in total at the moment, but word is on the street that more is on the way!
+
+## Inside the music pipeline
+
+While [mangomizer](https://osu.ppy.sh/users/1893718) and [pishifat](https://osu.ppy.sh/users/3178418) touched on the process of World Cup music production, how does it fare for other tournaments? **Kasumi** had this to say:
+
+> **Kasumi-sama:** The production process in GTS is very largely hands-off from us as admins. It all depends on the slot we shoot for, but for tiebreakers for example, we usually just give a general rundown of the structure we'd like, as well as a specific atmosphere we're shooting for. At that point, everything like genre, BPM and length, are basically decided by the artist's wish.
+> 
+> We only ask for relatively frequent drafts, in intervals of 30 seconds to 1 minute, so we can check if we first like the general vibe, as one would expect, and if we can map it accordingly to the planned difficulty, of course.
+
+The other tournament hosts interviewed here said basically the same thing, which means there's only one thing to do. (To ask some artists.)
```

Comment:
i feel like having `(To ask some artists.)` in brackets is like explaining a joke right after you said it, i don't think its necessary?

Reply:
maybe, although to me it doesn't really sound like a joke

(applied)